plc4x/throwing-the-towel.adoc

```diff
@@ -0,0 +1,227 @@
+
+=== My solution
+
+So I have decided and already announced in the project, that end of March 2024 I have stopped working on the project in my free time completely if I'm not compensated for it.
+
+2 donations I got recently were a direct result of me telling the people asking for help, that this is the way they are getting support from my side and I think both are quite happy with what they got.
+
+I have set up a private repo of my own, where I'm doing work on stuff that I need or simply want to do.
+Right now it contains a completely rewritten version of an Allen Bradley Logix driver that in contrast to the PLC4X-version supports auto-discovery, browsing and reading and writing of user-defined types.
+Also will all work on the UI client for PLC4X, that I already started in that repo.
+
+Things I implement there I do plan on donating to the open-source project, however I will only do that if I receive not insignificant amount of money with donations.
+As I don't run a company I can't sell anything.
+I know this is not the way the industry operates, but to be frank: I no longer care ... if they want my stuff now they need to abide to my rules.
+
+I might continue to address things in the open-source project in my free time in the future, but these will only be because I'm using PLC4X in my home automation system or possibly someone dropped a larger donation in https://github.com/sponsors/chrisdutz/[GitHub Sponsors] or https://buymeacoffee.com/christoferu[BuyMeACoffee] accounts.
+Anything beyond my previously mentioned 4 hours per week is now history.
+
+An Apache project usually is not a one-man-show. The project will definitely continue, however at a much slower pace (as you can probably see in the following commit statistic screenshot).
+
+image::commit-activity.jpg[]
+
+=== The Future of Apache PLC4X
+
+Things might change, as me stepping back possibly makes room for people that might have been kept from participating due to my over-proportional involvement.
+If everyone knows I'm covering all the bases, why bother?
+
+If that's the case, then things will normalize again.
+
+If however this doesn't happen, there also is a chance that the opposite could happen.
+Having nobody take care of the annoying house-work, and therefore not being able to concentrate on the fun parts, could also drive people away.
+
+In the case that community involvement dries up even more, Apache PLC4X could be aimed at the Apache Attic.
+This is the place where all Apache projects go, once the community around an Apache project is no longer able or willing to provide the amount of support that's expected of it.
+
+I will continue to invest my free time into open-source. However, just in projects that I believe have a sustainable future or just for fun for other projects which I have some other strange form of affection for.
```

Comment:
just for fun. After all there are other projects which I have affection for.
Ändert die Bedeutung vermutlich, der letzte Teilsatz hängt da aber etwas komisch dran.